webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/reports/ExpectedReports.scala

```diff
@@ -417,9 +443,22 @@ object ExpectedReportsSerialisation {
         // , (json \ "cardinality") // #10625: ignore cardinality
         , (json \ "values").extractOpt[List[String]]
         , (json \ "unexpanded").extractOpt[List[String]]
+        , (json \ "subComponents") match {
```

Comment:
Extractor and serializer don't have se same names ... I need to take care of this